scripts/configure-secrets.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh -e
+
+NOCOLOR='\033[0m'
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BOLD='\033[1m'
+NOBOLD='\033[21m'
+
+usage () {
+cat <<HELP_USAGE
+usage: $0 param1 param[n]...
+
+   Parameters:
+   - param1: The SDK name which needs to have a -Secrets.xcconfig file
+   - param[n]: a list of secrets environment variables to put in secret config file
+
+  Example usage:
+    $0 SignatureVerifier RSIGNATURE_VERIFIER_KEY_FETCH_ENDPOINT RAS_APPLICATION_IDENTIFIER RAS_SUBSCRIPTION_KEY
```

Comment:
I think this should either match the key used `RSV_KEY_FETCH_ENDPOINT` or be a generic example

Reply:
Ah I missed this